pkg/crc/version/version.go

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright (C) 2019 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package version
```

Comment:
I am not so sure if we need to push this so far into the package structure.
It seems to be more relatedto the actual build command? or overall library?

Reply:
@gbraad We need to include the logic or version anyway. Now or later? I would rather include this soon than later.

Reply:
@gbraad Also we should change the location of the pkg file to suit our code i.e. as our code evolves.

Reply:
It makes more sense to start at `pkg/crc/version.go` for this

Reply:
@gbraad What is upside for not having a directory for separate packages? IMO it should be standard practice rather than moving files in to separate directory when there are multiple files for a single package.

Reply:
It has current NO logical use.

Reply:
if you would actually use the code from the `root.go` we have a different situation.

Reply:
> standard practice rather than moving files in to separate directory when there are multiple files for a single package.

package is a logical grouping. I do not see how 'version' is now a logical grouping. It is also part of the something crc-specific... so, placing this in `crc` makes sense

Reply:
@gbraad Fixed it. Take a look.